core/chains/evm/client/errors.go

```diff
@@ -465,3 +465,73 @@ func NewSendErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction
 	}
 	return clienttypes.Unknown, err
 }
+
+// Solution to handle SendOnly nodes error codes. In that case, we don't assume there is another transaction that will be correctly
+// priced. Ideally in the future we wouldn't need an entire new function for this.
+func NewSendOnlyErrorReturnCode(err error, lggr logger.Logger, tx *types.Transaction, fromAddress common.Address, isL2 bool) (clienttypes.SendTxReturnCode, error) {
```

Comment:
This is not correct. If you are going to create a new function you should only include the functionalities you need. The rest of the parsing is not needed or used. Parse the message and categorize it as either Successful, TransactionAlreadyKnown, and the rest.

Reply:
fixed